test/test_tcp.rs

```diff
@@ -418,6 +418,13 @@ fn write_bufs() {
             match s.write_bufs(&b) {
                 Ok(n) => so_far += n,
                 Err(e) => {
+                    #[cfg(windows)]
```

Comment:
You do mean to check this only on Windows with debug assertions enabled? If so you can use `#[cfg(and(windows, not(debug_assertions)]`. I don't think any Unix platforms needs this check.

Reply:
Same thing happened on linux.
The common point is error ConnectionReset or ConnectionAbort will occur on Release channel's test